shared/src/enums/exception-message.enum.ts

```diff
@@ -2,7 +2,7 @@ enum ExceptionMessage {
     INVALID_CREDENTIALS = 'You entered an incorrect e-mail address or password',
     EMAIL_ALREADY_EXISTS = 'User with this email already exists',
     INVALID_TOKEN = 'Invalid token',
-    AUTHORIZE_ERROR = 'authorize error'
+    AUTHORIZE_ERROR = 'authorize error',
```

Comment:
```suggestion
    AUTHORIZE_ERROR = 'You should authorize',
```

Reply:
Resolved in [fe2799b](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/fe2799be923f4707b833676e199dd3c160b9de92)